solr/solrj/src/java/org/apache/solr/client/solrj/io/SolrClientCache.java

```diff
@@ -100,10 +100,11 @@ public synchronized CloudSolrClient getCloudSolrClient(String zkHost) {
   }
 
   @Deprecated(since = "9.0")
-  public synchronized HttpSolrClient getHttpSolrClient(String host) {
-    HttpSolrClient client;
+  // is this really deprecated?  what if it's getSolrClient?  It COULD be!
```

Comment:
The name is fine because the argument is in fact a URL.  But I agree it should not be deprecated.

Reply:
I'd appreciate docs on the argument if you have time.  It's possibly kind of wrong; appears to be baseUrl actually?

Reply:
i think this is a good follow up candidate...   I updated the host --> baseUrl, but I think we also should undeprecate these methods...    in another issue?

Reply:
in another issue is fine!